includes/Domain/Resources/McpResource.php

```
@@ -344,7 +344,8 @@ public function to_array(): array {
 	 * @return string
 	 */
 	public function to_json(): string {
-		return wp_json_encode( $this->to_array() );
+		$json = wp_json_encode( $this->to_array() );
+		return false !== $json ? $json : '{}';
```

Do we _want_ to fail silently, here?

---

Probably not. We can address this in a follow-up issue/PR